modules/page.py

```diff
@@ -13,17 +9,17 @@ class pageSkel( HierarchySkel ):
 	descr = textBone( descr="Content", required=True, searchable=True )
 
 class Page( Hierarchy ):
-	adminInfo = {	"name": "Sites", #Name of this modul, as shown in ViUR Admin (will be translated at runtime)
+	adminInfo = {	"name": "Sites", #Name of this module, as shown in ViUR Admin (will be translated at runtime)
 				"handler": "hierarchy",  #Which handler to invoke
-				"icon": "icons/modules/hierarchy.svg", #Icon for this modul
-				"formatstring": "$(name)", 
-				"filters" : { 	
```

Comment:
What is this for a piece of code? I know filters as a dict of filter criteria, but not with a key None containing a dict. 
I think this is a very obsolete definition of a views entry... can you please convert this to a current implementation, according to https://github.com/viur-framework/vi/wiki/adminInfo-in-List-modules#views ?